frontend/src/components/editor/chrome/panels/session-panel.tsx

```diff
@@ -0,0 +1,75 @@
+/* Copyright 2024 Marimo. All rights reserved. */
+
+import { DatabaseIcon, VariableIcon } from "lucide-react";
+import React, { useState } from "react";
+import { DataSources } from "@/components/datasources/datasources";
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import { Badge } from "@/components/ui/badge";
+import { VariableTable } from "@/components/variables/variables-table";
+import { useCellIds } from "@/core/cells/cells";
+import { useDatasets } from "@/core/datasets/state";
+import { useVariables } from "@/core/variables/state";
+
+const SessionPanel: React.FC = () => {
+  const variables = useVariables();
+  const cellIds = useCellIds();
+  const datasets = useDatasets();
+  const [openSections, setOpenSections] = useState<string[]>(["variables"]);
```

Comment:
+1 to store this in localstorage. i (n=1) use datasources much more than the variables